core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -125,14 +126,16 @@ export class ExecuteWorkflowService {
         Object.entries(event.generalErrors).forEach(entry => {
           errorMessages[entry[0]] = entry[1];
         });
-        return {state: ExecutionState.Failed, errorMessages: errorMessages};
+        return { state: ExecutionState.Failed, errorMessages: errorMessages };
       // TODO: Merge WorkflowErrorEvent and ErrorEvent
       case 'WorkflowExecutionErrorEvent':
         const backendErrorMessages: Record<string, string> = {};
         Object.entries(event.errorMap).forEach(entry => {
           backendErrorMessages[entry[0]] = entry[1];
         });
-        return {state: ExecutionState.Failed, errorMessages: backendErrorMessages};
+        return { state: ExecutionState.Failed, errorMessages: backendErrorMessages };
+      case 'PythonPrintTriggeredEvent':
+        return { state: ExecutionState.Running, consoleMessages: event };
```

Comment:
i don't think you need to include it in here, just leave it in `default`, most messages handled here are affecting the state in a way, messages that don't affect state (such as `ResultUpdate` simply doesn't affect the state)

Reply:
removed.